src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1050,7 +1060,7 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                         $prefix = ''
                                         $newline = [Environment]::Newline
 
-                                        if ($myinv -and $myinv.ScriptName -or $myinv.ScriptLineNumber -gt 1 -or $_.CategoryInfo.Category -eq 'ParserError') {
+                                        if ($myinv -and $myinv.ScriptName -or $myinv.ScriptLineNumber -gt 1 -or $err.CategoryInfo.Category -eq 'ParserError') {
```

Comment:
just ParserError? not `@('ParserError') -contains $err.CategoryInfo.Category`?

Reply:
From my experimentation, only ParserError needs to be covered here as it provides additional context on the location within the line of the issue.